code/datums/helper_datums/command_alerts.dm

```diff
@@ -20,9 +20,11 @@
 	force_report = 1
 
 	var/level = 1
+	var/level_max = 7
```

Comment:
It would be neat if these actually related to the disease somehow, since we have badness, intensity, spread danger expressed as a bitflag value. Calculate up a real threat level.

Reply:
I agree but biohazard alerts are actually used in many places in the code and for many unrelated reasons (such as the organ failure random event). So it's more the alert itself needing a rewrite, but it's not essential for nuviro so I'm postponing that.